docbook/Admin_Guide/en-US/config/auth.xml

```diff
@@ -231,6 +231,16 @@ ldaps://ldap.example.com:3269/
 					the data retrieved from LDAP when ON.</para>
 				</listitem>
 			</varlistentry>
+			<varlistentry>
+				<term>$g_ldap_cache_fields</term>
+				<listitem>
+					<para>Choose witch fields will be add to the cache for
+						a user from LDAP. $g_ldap_cache_fields =
+						array( 'mail', 'roomNumer' );
```

Comment:
It may be worth adding that such extra LDAP fields can be used from plugins.

Reply:
I also suggest remove `mail` from the example since it is built in.

Reply:
Suggestion added in comment.
`mail` removed but others potentials fields  added to see a real array.